gradle-jdks-distributions/src/main/java/com/palantir/gradle/jdks/JdkDistributionName.java

```diff
@@ -24,7 +24,8 @@
 public enum JdkDistributionName {
     AZUL_ZULU,
     AMAZON_CORRETTO,
-    GRAALVM_CE;
+    GRAALVM_CE,
+    LOOM_EA;
```

Comment:
Rather than loom EA specifically, can we just have one for all JDK EAs (and have loom in the version string?)

Reply:
Looks like it is in the version string? `24-loom+7-60`

Reply:
Yeah I think I can make this work. The URLs are ever so slightly different for e.g. [JDK24 EA builds](https://download.java.net/java/early_access/jdk24/17/GPL/openjdk-24-ea+17_linux-aarch64_bin.tar.gz) versus [project-specific builds](https://download.java.net/java/early_access/loom/7/openjdk-24-loom+7-60_linux-aarch64_bin.tar.gz), but I think we can work around that.

Reply:
@CRogers i think https://github.com/palantir/gradle-jdks/pull/423/commits/e0fb0db5a32b2bef1b6fc0091892bd2233fd4795 should do the trick